lib/src/sql/statements/show.rs

```diff
@@ -0,0 +1,189 @@
+use crate::ctx::Context;
+use crate::dbs::Options;
+use crate::err::Error;
+use crate::sql::comment::shouldbespace;
+use crate::sql::common::take_u64;
+use crate::sql::error::IResult;
+use crate::sql::table::{table, Table};
+use crate::sql::value::Value;
+use derive::Store;
+use nom::branch::alt;
+use nom::bytes::complete::tag_no_case;
+use nom::character::complete::u32;
+use nom::combinator::map;
+use nom::combinator::opt;
+use nom::combinator::recognize;
+use nom::sequence::preceded;
+use serde::{Deserialize, Serialize};
+use std::fmt;
+
+// ShowStatement is used to show changes in a table or database via
+// the SHOW CHANGES statement.
+#[derive(Clone, Debug, Default, Eq, PartialEq, Serialize, Deserialize, Store, Hash)]
+pub struct ShowStatement {
+	pub table: Option<String>,
```

Comment:
Out of interest, is there a specific reason why `Option<Table>` isn't used? Not saying it needs to be...

Reply:
Not really... I vaguely remember that I'd just use String as I'd use it as literally a string had no need for other functions of the Table struct.

Assuming there's zero overhead in the encoded bytes and almost zero overhead in encoding/decoding process, we could just use Table for consistency!

578957ea